packages/workbox-cli/src/lib/errors.js

```diff
@@ -24,4 +24,14 @@ module.exports = {
   'config-validation-failed': `Your configuration is invalid:`,
   'workbox-build-runtime-error': `Service worker generation failed:`,
   'unknown-command': `Unknown command:`,
+  'no-file-extensions-found': ol`No files could be found that are suitable for
+    caching.`,
+  'no-file-extensions-selected': `Please select at least one file extension.`,
+  'sw-src-does-not-exist': ol`The source service worker file you provided does
+    not exist. Please provide the path to an existing file.`,
+  'invalid-sw-dest': ol`Please enter a valid path to use for the service worker
```

Comment:
Since this is the destination path, what exactly needs to the exist, the directory? Or something else? The error message makes it sound like I need to create an empty file.

Reply:
It's an error message that's displayed when you provide an empty string as the `swDest` value.

If you provide any other value, you won't see that error message, but you may run into runtime errors when you generate a SW if the path can't be used.